### PR TITLE
Cw 55 allow saving templateswith fiat amount

### PR DESCRIPTION
**Pull request by Serhii-Borodenko**
Added ability to save templates with fiat amount

### CODE REVIEW

File: lib/src/screens/send/send_page.dart [hunk 1]
```diff
@@ -49,6 +53,13 @@ class SendPage extends BasePage {
   @override
   AppBarStyle get appBarStyle => AppBarStyle.transparent;
 
+  @override
+  void onClose(BuildContext context) {
+    settingsViewModel.setFiatCurrency(fiatFromSettings);
+    Navigator.of(context).pop();
+    return print(fiatFromSettings.title);
+  }
+
```
Comment (mkyq) [inline lib/src/screens/send/send_page.dart:62]: What that code should do ?

**Review by mkyq — CHANGES_REQUESTED** [review]

**Comment by mkyq** [comment]
cc @Serhii-Borodenko

File: lib/src/screens/send/send_page.dart (hunk 1 above)
Comment (Serhii-Borodenko) [reply lib/src/screens/send/send_page.dart]: Removed print statement

Comment (Serhii-Borodenko) [reply lib/src/screens/send/send_page.dart]: @mkyq

**Review by mkyq — APPROVED** [review]

**Pull request merged**